src/desugar.rs

```diff
@@ -422,8 +422,8 @@ pub(crate) fn desugar_command(
             desugared.push(NormCommand::Fail(Box::new(last.unwrap())));
             desugared
         }
-        Command::Input { .. } => {
-            todo!("desugar input");
```

Comment:
@oflatt Is there a potential pitfall here that prevents you from just building a `NormCommand::Input`?

Reply:
I think I just didn't understand what the input command did lol, thanks for the fix